frontend/pages/UserSettingsPage/UserSettingsPage.jsx

```diff
@@ -332,10 +345,11 @@ export class UserSettingsPage extends Component {
           </p>
           <Button
             onClick={onShowApiTokenModal}
-            className={`${baseClass}__button button button--grey`}
+            className={`${baseClass}__button`}
           >
             Get API token
           </Button>
+          <span className={`${baseClass}__version`}>{`Version: Fleet ${version.version} • Go ${version.go_version}`}</span>
```

Comment:
Perhaps looks a little cleaner this way?

```suggestion
          <span className={`${baseClass}__version`}>{`Fleet ${version.version} • Go ${version.go_version}`}</span>
```

Reply:
Agreed ✨